src/Bicep.LangServer/LangServerConstants.cs

```diff
@@ -6,6 +6,7 @@ namespace Bicep.LanguageServer
     public static class LangServerConstants
     {
         public const string BuildCommand = "build";
+        public const string GenerateParamsCommand = "generateParams";
```

Comment:
Please add a test for this new command - you could probably base this off https://github.com/Azure/bicep/blob/7d2b7818e0bd0432560f46713a8584dc21fa12bc/src/Bicep.LangServer.IntegrationTests/BuildCommandTests.cs#L22-L23

Reply:
done! added a test to check if generate-params command works as expected 👍